task1/src/csv.js

```diff
@@ -0,0 +1,48 @@
+import { appendFileSync, existsSync, unlinkSync } from 'fs';
+import { csv } from 'csvtojson';
+
+const csvFilePath = './csv/input.csv';
+const txtFilePath = './txt/output.txt';
+
+deleteTxtFile(txtFilePath);
+convertCsvData();
+
+function convertCsvData() {
+  csv()
+    .fromFile(csvFilePath)
+    .subscribe(
+      (jsonData) => writeJsonToTxt(jsonData),
+      (err) => console.error('Error:', err),
+      () => console.log('Success')
+    );
+};
+
+function writeJsonToTxt(jsonData) {
+  appendFileSync(txtFilePath, prepareJson(jsonData), function (err) {
```

Comment:
As it was mentioned on our Q&A session it's better to use async methods of working with files.

Reply:
appendFileSync was changed to async method appendFile